userspace/libsinsp/utils.cpp

```diff
@@ -958,7 +958,7 @@ void sinsp_utils::ts_to_string(uint64_t ts, OUT string* res, bool date, bool ns)
 void sinsp_utils::ts_to_iso_8601(uint64_t ts, OUT string* res)
 {
 	static const char *fmt = TS_STR_FMT;
-	char buf[sizeof(fmt) + 1];
+	char buf[strlen(fmt)];
```

Comment:
```suggestion
	char buf[strlen(fmt)+1];
```

Or maybe `char buf[sizeof(TS_STR_FMT)]` but I don't have a strong opinion either way

Reply:
Let's go with `sizeof`